src/twisted/conch/test/test_keys.py

```diff
@@ -564,6 +612,14 @@ def test_notEqual(self):
         self.assertTrue(rsa1 != None)
 
 
+    def test_dataError(self):
+        """
+        The data() method raises RuntimeError for bad keys.
```

Comment:
Minor nitpick: we usually use epytext syntax to do references in test names, so rather than the ambiguous "the data() method", we'd say, `L{keys.Key.data} raises RuntimeError for bad keys`.